apps/gallery/js/pick.js

```diff
@@ -220,12 +220,14 @@ var Pick = (function() {
     // then we use null as the type. This value is passed to
     // cropResizeRotate() and will leave the image unchanged if possible
     // or will use jpeg if changes are needed.
+    // EXIF should be preserved.
```

Comment:
You're handling the pick case here, but don't you also need to handle share?  See frames.js:shareSingleItem()
Or file a followup bug for this case.